docs/customservices.md

```diff
@@ -141,3 +140,20 @@ See https://docs.portainer.io/v/ce-2.11/user/account-settings#access-tokens
   type: "Portainer"
   apikey: "MY-SUPER-SECRET-API-KEY"
 ```
+
+## Uptime Kuma
+
+Using the Uptime Kuma service you can display info about your instance uptime right on your Homer dashboard.
+
+The following configuration is available for the UptimeKuma service. Needs v1.13.1 or later because of the change in APIs due to [multiple status pages support](https://github.com/louislam/uptime-kuma/releases/tag/1.13.1).
+
+As usual, needs CORS to be enabled from the homer domain to uptime kuma.
```

Comment:
```suggestion
```

Another small nit: we don't mention CORS anywhere else on this page, and in general, we're looking to address the CORS issue with better documentation and potentially a first-party CORS proxy soon (TM)

Reply:
Ooo that's exciting! Looking forward to that.